lms/djangoapps/teams/static/teams/js/spec/views/instructor_tools_spec.js

```diff
@@ -29,29 +35,28 @@ define([
                     $('.action-secondary').click();
                 }
             },
-            expectSuccessMessage = function (team) {
-                expect(TeamUtils.showMessage).toHaveBeenCalledWith(
-                    'Team "' + team.get('name') + '" successfully deleted.',
-                    'success'
+            expectSuccessMessage = function(view) {
+                var team = view.team;
+                expect($('.msg-content .copy').text().trim()).toContain(
```

Comment:
You almost always need trim on text that you take out of the DOM. I'l try it without, but I wouldn't expect it to work.